TUTORIAL.md

```diff
@@ -20,9 +20,9 @@ AssertPathIsDirectory=/mnt/plexdrive
 After=network-online.target
 
 [Service]
-Type=simple
+Type=notify
 ExecStart=/usr/bin/plexdrive mount -v 2 /mnt/plexdrive
-ExecStop=/bin/fusermount -u /mnt/plexdrive
+ExecStopPost=-/bin/fusermount -quz /mnt/plexdrive
```

Comment:
The `-` at the beginning makes the command optional, the `-q` option prevents noise on clean shutdown when there is nothing to unmount and `-z` is lazy unmount, which allows the unmount to succeed as soon as there are no more open file handles.